site/content/docs/main/guides/grpc.md

```diff
@@ -144,10 +144,43 @@ spec:
 
 ## Gateway API Configuration
 
-At the moment, configuring gRPC routes with Gateway API resources is achieved by the same method as with Ingress v1: annotation to select a protocol and port on a Service referenced by HTTPRoute `spec.rules[].backendRefs`.
+Gateway API now supports a specific resource [GRPCRoute][6] for routing gRPC requests.
 
-Gateway API does include a specific resource [GRPCRoute][6] for routing gRPC requests.
-This may be supported in future versions of Contour.
+## GRPCRoute Configuration
```

Comment:
I think we can drop this section header, this content can all go directly under "Gateway API Configuration"